docs/etcd/demo_cluster_setup.md

```diff
@@ -1,14 +1,42 @@
+**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production. We strongly encourage you to follow more thorough guides if you want to use etcd in production.
 
-In the first part, *Create the etcd cluster*, we will install and start etcd. The second part, *Make it more secure*, is not mandatory but **highly** suggested.
+The first section focuses on installing etcd in the same Kubernetes cluster as the operator. For a standalone installation jump to *Create the etcd cluster*. The section *Make it more secure*, is not mandatory but **highly** suggested.
 
-**IMPORTANT NOTE**: this guide will only help you create a **demo** cluster so that you can quickly have a working example to use with the CN-WAN Operator and is not intended to be used in production. We strongly encourage you to follow more thorough guides if you want to use etcd in production.
+## Quick install on Kubernetes
+
+Since the operator runs on Kubernetes, it's safe to assume you have a Kubernetes cluster available, and the simplest way to get up and running quickly with etcd is to install it on the same Kubernetes cluster where you will run the operator. Please note that this etcd installation is separate from the etcd service that comes bundled with Kubernetes.
+
+If you prefer to use a binary on your favorite OS or docker instead, jump to the next section.
+
+We will use *Helm* to install etcd on Kubernetes. The `bitnami` Helm repo holds a chart that is well maintained, so let's add that repo:
+
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    helm repo update
+
+Check customization options for the `bitnami/etcd` chart:
+
+    helm show values bitnami/etcd | less
+
+Custom options can be added to a `values.yaml` file, or specified on the command line. To install etcd on a Kubernetes cluster in a public cloud using a load balancer and authentication enabled, run this:
+
+    kubectl create namespace etcd
+    helm install etcd bitnami/etcd --namespace etcd --set service.type=LoadBalancer,auth.rbac.enabled=true,auth.rbac.rootPassword=dem0-pwd
+
+The notes printed after the helm chart installation are very thorough on how to use your shiny new etcd installation. First, you should run a pod with the etcdctl client client:
+
+    kubectl run etcd-client --restart='Never' --image docker.io/bitnami/etcd --env ROOT_PASSWORD=$(kubectl get secret --namespace etcd etcd -o jsonpath="{.data.etcd-root-password}" | base64 --decode) --env ETCDCTL_ENDPOINTS="etcd.etcd.svc.cluster.local:2379" --namespace etcd --command -- sleep infinity
+
+And then create a local shell alias for the etcdctl command on the pod:
+
+    alias etcdctl="kubectl exec -it etcd-client -n etcd -- etcdctl --user root:$(kubectl get secret --namespace etcd etcd -o jsonpath="{.data.etcd-root-password}" | base64 --decode)"
+
```

Comment:
*very optional*:

Now try to insert value "test" with key `/test`:

```bash
etcdctl put /test "test"
```

Now try to read it

```bash
etcdctl get /test
```